example/spec/redeployment/LambdaVersionOption.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const fs = require('fs-extra');
+const jsyaml = require('js-yaml');
+
+const { aws: { cf, sfn } } = require('@cumulus/common');
+
+const { getWorkflowArn } = require('@cumulus/integration-tests');
+const {
+  loadConfig,
+  redeploy
+} = require('../helpers/testUtils');
+
+const config = loadConfig();
+
+async function cleanUp() {
+  console.log('Cleaning up deployment');
+  await fs.remove('test_app');
+  await redeploy(config);
+}
+
+describe('When the useWorkflowLambdaVersions option is set to false the deployment', () => {
+  let workflowDefinitions;
+  let stackList;
+  const deletedStatuses = ['DELETE_COMPLETE'];
+  const lambdaVersionStackName = `${config.stackName}-WorkflowLambdaVersionsNestedStack`;
+  const startDate = new Date();
+
+  beforeAll(async () => {
+    try {
+      // ensure we're working with a fresh deployment
+      await redeploy(config);
+
+      //Flip the config value
+      await fs.copy('node_modules/@cumulus/deployment/app', 'test_app');
+      const configString = await fs.readFile('./test_app/config.yml', 'utf-8');
+      const updatedConfig = configString.replace(/useWorkflowLambdaVersions: true/, 'useWorkflowLambdaVersions: false/');
+      await fs.writeFile('./test_app/config.yml', updatedConfig, 'utf-8');
+
+      //Redeploy with custom configuration
+      await redeploy(config, 'test_app/', 'node_modules/@cumulus/deployment/app/kes.js');
```

Comment:
I like this redeployment strategy but wouldn't the test run faster if we stub `this.config.useWorkflowLambdaVersions` in the `packages/deployment/app/kes.js`?

Reply:
Yes this is a non-blocking comment.